common/app/common/configuration.scala

```diff
@@ -440,10 +440,6 @@ class GuardianConfiguration extends Logging {
     lazy val expiredPaidContentUrl = s"${site.host}/info/2015/feb/06/paid-content-removal-policy"
   }
 
-  object open {
-    lazy val ctaApiRoot = configuration.getMandatoryStringProperty("open.cta.apiRoot")
```

Comment:
@TBonnin Good catch. Deleted in https://github.com/guardian/frontend/pull/17404/commits/da23db2ec8d1a213f10b097dfdc1ee237c33a30e